tools/interactive/interactivetool_jupyter_notebook.xml

```diff
@@ -17,12 +17,22 @@
     </environment_variables>
     <command detect_errors="aggressive"><![CDATA[
         #import re
+        #import random
+        #import string
         export GALAXY_WORKING_DIR=`pwd` &&
         mkdir -p ./jupyter/outputs/ &&
         mkdir -p ./jupyter/data &&
 
-        #set $cleaned_name = re.sub('[^\w\-\.]', '_', str($input.element_identifier))
-        ln -sf '$input' './jupyter/data/${cleaned_name}' &&
+        #set $cleaned_names = []
+        #for $file in $input
```

Comment:
you could use enumerate() here and then get a unique number?

Reply:
Do we want to have the order of inputs be significant though?

Reply:
I changed the names to be prefixed with an index according to your suggestion.